testing/microlith-test/Dockerfile

```diff
@@ -16,81 +16,80 @@ RUN apk --no-cache add \
 
 # Install Detik - we use cd to expand the variables
 RUN curl -LO "https://github.com/bats-core/bats-detik/archive/refs/tags/v$BATS_DETIK_VERSION.zip" && \
-	unzip -q -d /tmp "v$BATS_DETIK_VERSION.zip" && \
+    unzip -q -d /tmp "v$BATS_DETIK_VERSION.zip" && \
     mkdir -p /usr/local/lib/bats-detik/ && \
     cd "/tmp/bats-detik-$BATS_DETIK_VERSION" && \
-	mv -f lib/* /usr/local/lib/bats-detik/ && \
-	rm -rf "/tmp/bats-detik-$BATS_DETIK_VERSION" && \
+    mv -f lib/* /usr/local/lib/bats-detik/ && \
+    rm -rf "/tmp/bats-detik-$BATS_DETIK_VERSION" && \
     rm -f "v$BATS_DETIK_VERSION.zip"
 
 RUN curl -LO "https://github.com/bats-core/bats-assert/archive/refs/tags/v$BATS_ASSERT_VERSION.zip" && \
-	unzip -q -d /tmp "v$BATS_ASSERT_VERSION.zip" && \
+    unzip -q -d /tmp "v$BATS_ASSERT_VERSION.zip" && \
     mkdir -p /usr/local/lib/bats-assert/ && \
     cd "/tmp/bats-assert-$BATS_ASSERT_VERSION" && \
-	mv -f * /usr/local/lib/bats-assert/ && \
-	rm -rf "/tmp/bats-assert-$BATS_ASSERT_VERSION" && \
+    mv -f * /usr/local/lib/bats-assert/ && \
+    rm -rf "/tmp/bats-assert-$BATS_ASSERT_VERSION" && \
     rm -f "v$BATS_ASSERT_VERSION.zip"
 
 RUN curl -LO "https://github.com/bats-core/bats-support/archive/refs/tags/v$BATS_SUPPORT_VERSION.zip" && \
-	unzip -q -d /tmp "v$BATS_SUPPORT_VERSION.zip" && \
+    unzip -q -d /tmp "v$BATS_SUPPORT_VERSION.zip" && \
     mkdir -p /usr/local/lib/bats-support/ && \
     cd "/tmp/bats-support-$BATS_SUPPORT_VERSION" && \
-	mv -f * /usr/local/lib/bats-support/ && \
-	rm -rf "/tmp/bats-support-$BATS_SUPPORT_VERSION" && \
+    mv -f * /usr/local/lib/bats-support/ && \
+    rm -rf "/tmp/bats-support-$BATS_SUPPORT_VERSION" && \
     rm -f "v$BATS_SUPPORT_VERSION.zip"
 
 RUN curl -LO "https://github.com/bats-core/bats-file/archive/refs/tags/v$BATS_FILE_VERSION.zip" && \
-	unzip -q -d /tmp "v$BATS_FILE_VERSION.zip" && \
+    unzip -q -d /tmp "v$BATS_FILE_VERSION.zip" && \
     mkdir -p /usr/local/lib/bats-file/ && \
     cd "/tmp/bats-file-$BATS_FILE_VERSION" && \
-	mv -f * /usr/local/lib/bats-file/ && \
-	rm -rf "/tmp/bats-file-$BATS_FILE_VERSION" && \
+    mv -f * /usr/local/lib/bats-file/ && \
+    rm -rf "/tmp/bats-file-$BATS_FILE_VERSION" && \
     rm -f "v$BATS_FILE_VERSION.zip"
 
 ENV BATS_DETIK_ROOT=/usr/local/lib/bats-detik
 ENV BATS_FILE_ROOT=/usr/local/lib/bats-file
 ENV BATS_SUPPORT_ROOT=/usr/local/lib/bats-support
 ENV BATS_ASSERT_ROOT=/usr/local/lib/bats-assert
 
-# Use the same UID as Jenkins:
-# For Jenkins versions < 2.62, this is 1000
-RUN adduser -D -u 10000 testing
-USER testing
+COPY . /home/testing/
 WORKDIR /home/testing
 
-COPY --chown=testing . ./
-
 # This is the image we want to test against, override to change.
-ENV COS_IMAGE=couchbase/observability-stack:v1
-ENV COUCHBASE_SERVER_IMAGE=couchbase/server:6.6.2
+ARG CMOS_IMAGE=couchbase/observability-stack:v1
+ENV CMOS_IMAGE=${CMOS_IMAGE}
+ARG CMOS_PORT=8080
+ENV CMOS_PORT=${CMOS_PORT}
+ARG COUCHBASE_SERVER_IMAGE=couchbase/server:7.0.1
```

Comment:
May be worth defaulting to 6.x given that's the target for 0.1

Reply:
Agreed.